dining/views.py

```diff
@@ -0,0 +1,86 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from django.http import JsonResponse, HttpResponseForbidden
+from datetime import timedelta, datetime
+
+from legacy.models import Account, DiningBalance, DiningTransaction, Course
+from options.models import get_value, get_option
+from studentlife.utils import get_new_start_end
+
+# Create your views here.
+
+class Dashboard(APIView):
+
+	def get(self, request, format=None):
+		if not self.request.user.is_authenticated:
+			return HttpResponseForbidden()
+
+		self.pennid = "76627463" # request.user.username
+		self.uid = Account.objects.filter(pennid=self.pennid)[0].id
+		json = self.balance()
+
+		start, end = self.get_semester_start_end()
+		json["start-of-semester"] = start
+		json["end-of-semester"] = end
+
+		json["cards"] = []
+		json["cards"].append(self.recent_transactions_card())
```

Comment:
```suggestion
		json["cards"] = [self.recent_transactions_card()]
```
and delete the line above to do it in one line

Reply:
Thank so much, will do all of that. There are also a few things that Dom has requested be added quickly, so I'll add those in to and commit again.